viteup/src/utils.ts

```diff
@@ -1,3 +1,20 @@
+import { SUPPORTED_EXPORT_CONDITIONS } from "./types";
+
 export function isObject(obj: unknown) {
   return typeof obj === "object" && obj !== null;
 }
+
+export function shouldSkipExport(exportPath: string) {
+  const bar = exportPath.split(/^\.?\//, 2);
+  if (bar[1] && bar[1].indexOf(".") === -1) {
+    // check directory
+    if (exportPath.slice(-1) === "/") {
+      return true;
+    }
+  } else if (exportPath !== ".") {
```

Comment:
This is the first thing we should check for in order to skip some logic when the exportPath is `.`